dev/WindowsAppRuntime_BootstrapDLL/WindowsAppRuntime.Bootstrap.rc

```diff
@@ -0,0 +1,33 @@
+#include <winresrc.h>
+
+#define STR1(x) #x
+#define DELIM STR1(.)
+#define STR2(a,b)     STR1(a) DELIM STR1(b)
+
+#define WAS_RELEASE_MAJOR           1
+#define WAS_RELEASE_MINOR           2
+#define VER_PRODUCT_VERSION         WAS_RELEASE_MAJOR, WAS_RELEASE_MINOR, 0, 0
+#define VER_PRODUCT_VERSION_STRING  STR2(WAS_RELEASE_MAJOR, WAS_RELEASE_MINOR)
+
+VS_VERSION_INFO VERSIONINFO
+ PRODUCTVERSION VER_PRODUCT_VERSION
+ FILEFLAGSMASK 0x3fL
+BEGIN
+    BLOCK "StringFileInfo"
+    BEGIN
+        BLOCK "040904b0"
+        BEGIN
+            VALUE "CompanyName", "Microsoft Corporation"
+            VALUE "FileDescription", "WindowsAppRuntime.Bootstrap.dll"
+            VALUE "InternalName", "WindowsAppRuntime.Bootstrap.dll"
+            VALUE "LegalCopyright", "Copyright (c) Microsoft Corporation. All rights reserved."
+            VALUE "OriginalFilename", "WindowsAppRuntime.Bootstrap.dll"
+            VALUE "ProductName", "Windows App SDK"
+            VALUE "ProductVersion", VER_PRODUCT_VERSION_STRING "\0"
+        END
+    END
+    BLOCK "VarFileInfo"
+    BEGIN
+        VALUE "Translation", 0x409, 1200
```

Comment:
If there's no variable info is this BLOCK needed at all?

Remove?

Reply:
Unfortunately yes.  The translation fields says what language to make the file info.  Without it, nothing shows up.